src/Trixi.jl

```diff
@@ -52,7 +52,7 @@ using P4est
 using Setfield: @set
 using RecipesBase: RecipesBase
 using Requires: @require
-using Static: Static, One
+using Static: Static, One, True, False
```

Comment:
`using True, False` will automatically make someone who is `using Trixi` allow to use `Trixi.True()` and `Trixi.False()`?

Reply:
Yes

Reply:
Thanks!